src/core/src/OmniPolygonRasterOverlay.cpp

```diff
@@ -85,7 +85,20 @@ void OmniPolygonRasterOverlay::reload() {
         return;
     }
 
-    const auto invertSelection = false;
+    auto invertSelection = false;
+    auto& path = getPath();
+    auto& usdStage = _pContext->getUsdStage();
+    const auto prim = usdStage->GetPrimAtPath(path);
+    if (prim.IsValid()) {
+        const auto cesiumPolygonRasterOverlay = UsdUtil::getCesiumPolygonRasterOverlay(_pContext->getUsdStage(), _path);
+        auto attr = cesiumPolygonRasterOverlay.GetInvertSelectionAttr();
+        if (attr.IsValid()) {
+            bool value;
+            attr.Get(&value);
+            invertSelection = value;
+        }
+    }
```

Comment:
Could you add a member function for this? See `OmniRasterOverlay::getShowCreditsOnScreen` for an example. Some things like checking prim and attr validity may not be needed.